scripts/archivematica_bags_migration.py

```diff
@@ -0,0 +1,212 @@
+from elasticsearch import Elasticsearch
+from elasticsearch import helpers
+import boto3
+import json
+import sys
+import os
+import shutil
+from tqdm import tqdm
+
+# from wellcome_storage_service import StorageServiceClient
+
+
+def get_aws_resource(resource, *, role_arn):
+    assumed_role_object = sts_client.assume_role(
+        RoleArn=role_arn, RoleSessionName="AssumeRoleSession1"
+    )
+    credentials = assumed_role_object["Credentials"]
+    return boto3.resource(
+        resource,
+        aws_access_key_id=credentials["AccessKeyId"],
+        aws_secret_access_key=credentials["SecretAccessKey"],
+        aws_session_token=credentials["SessionToken"],
+    )
+
+
+def get_aws_client(resource, *, role_arn):
+    assumed_role_object = sts_client.assume_role(
+        RoleArn=role_arn, RoleSessionName="AssumeRoleSession1"
+    )
+    credentials = assumed_role_object["Credentials"]
+    return boto3.client(
+        resource,
+        aws_access_key_id=credentials["AccessKeyId"],
+        aws_secret_access_key=credentials["SecretAccessKey"],
+        aws_session_token=credentials["SessionToken"],
+    )
```

Comment:
Note: our pattern (at your suggestion IIRC) is to put one-off scripts like this in the `migrations` folder. Once you're there, stuff like this is available to import from `common.py`.